helm/csi-powerstore/templates/_helpers.tpl

```diff
@@ -4,31 +4,31 @@ Return the appropriate sidecar images based on k8s version
 {{- define "csi-powerstore.attacherImage" -}}
   {{- if eq .Capabilities.KubeVersion.Major "1" }}
     {{- if and (ge (trimSuffix "+" .Capabilities.KubeVersion.Minor) "21") (le (trimSuffix "+" .Capabilities.KubeVersion.Minor) "24") -}}
```

Comment:
I think we should change these conditions to `le 1.25` otherwise for 1.24.x this check might fail

Reply:
@harishp8889 1.24.x, 24 is the minor version which is being checked here. So for any k8s1.24.x, the condition will pass. We are just checking the minor, not the patch